### PR TITLE
Add default MAC Address randomization security event check

### PR DESCRIPTION
**Pull request by lzammit**
# Description
Adding a MAC Address randomization security event and ask the user to disable that feature.

## Enhancements
* Add default mac randomization security event check

### CODE REVIEW

**Comment by julsemaan** [comment]
As per the description in #5755, we need to have basic instructions for Android and Apple devices on how to disable it.

File: html/captive-portal/templates/security_events/random_mac.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="u-padding">
+<div class="u-padding">
+<div class="o-layout--center">
+  <h5>Random MAC Address detection</h5>
```
Comment (extrafu) [inline html/captive-portal/templates/security_events/random_mac.html]: MAC address

File: html/captive-portal/templates/security_events/random_mac.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+</div>
+
+<div class="o-media o-media--error u-padding u-margin-bottom">
+  <div class="o-media__img">[% flashIcon(level='error') %]</div>
+  <p class="o-media__body">You have been detected using a random MAC Address, please disable that feature in your WIFI configuration on your device.</p>
```
Comment (extrafu) [inline html/captive-portal/templates/security_events/random_mac.html]: MAC Address -> MAC address
WIFI -> WiFi

File: html/captive-portal/templates/security_events/random_mac.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+</div>
+</div>
+
+<div class="o-layout--center">
+  <h6>Android</h6>
+  <img src="/content/images/android-mac-address-randomization.png"><img/>
+</div>
+
+<div class="o-layout--center">
+  <h6>iOS</h6>
```
Comment (extrafu) [inline html/captive-portal/templates/security_events/random_mac.html]: iOS -> Apple iOS

File: html/captive-portal/templates/security_events/random_mac.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  <img src="/content/images/ios-mac-address-randomization.jpg"><img/>
+</div>
+
+<div class="o-layout--center">
+  <h6>Windows 10</h6>
```
Comment (extrafu) [inline html/captive-portal/templates/security_events/random_mac.html]: Windows 10 -> Microsoft Windows 10

**Pull request merged**